docarray/typing/tensor/audio/abstract_audio_tensor.py

```diff
@@ -0,0 +1,25 @@
+from abc import ABC, abstractmethod
+from typing import BinaryIO, TypeVar, Union
+
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+
+T = TypeVar('T', bound='AbstractAudioTensor')
+
+
+class AbstractAudioTensor(AbstractTensor, ABC):
+    @abstractmethod
```

Comment:
rename to AudioTensorMixin and remove the abstract method. Move the common logic of save_to_wav_file in the other classes to this one.
For the logic related to converting tensor to bytes, you use the method `self.tobytes()` which we should define for tensor classes